.template/addons/github/.github/workflows/README.md.tt

```diff
@@ -9,10 +9,13 @@
 
 ### Secrets
 
-Setup the following secrets in order to run the workflows:
+Depends on the workflow, setup the following secrets:
```

Comment:
I don't quite get what the instructions are 😅  Do you mean:

```suggestion
Depending on the workflow used in the project, set up the following secrets:
```

Reply:
that's correct. Updated in 30ac803 🙈